src/Reanimate/Signal.hs

```diff
@@ -91,3 +92,16 @@ oscillateS t =
 --   <<docs/gifs/doc_bellS.gif>>
 bellS :: Double -> Signal
 bellS steepness = curveS steepness . oscillateS
+
+-- | Cubic Bezier signal. Gives you a fair amount of control over how the
+--   signal will 'curve'.
+--
+--   Example:
+--
+--   > signalA (cubicBezierS (0.0, 0.8, 0.9, 1.0)) drawProgress
+--   
+--   <<docs/gifs/doc_cubicBezierS.gif>>
+cubicBezierS :: Double -> Signal
```

Comment:
Looks like you meant: cubicBezierS :: (Double, Double, Double, Double) -> Signal